framework/test/unit/modules/token/init_genesis_state_fixture.ts

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+// import { getRandomBytes } from '@liskhq/lisk-cryptography';
+
+const oneLisk = BigInt('100000000');
```

Comment:
maybe `oneLiskToBeddows` is more accurate?

Reply:
I think `oneLSKToBeddows` is more accurate, since `Lisk` a brand and `LSK` is the token's name. It's also aligned with our functions like `convertLSKToBeddows` and we can use this function here like `const oneLSKToBeddows = convertLSKToBeddows('1');`, I think this is more readable.

Reply:
I think we could use `convertLSKToBeddows` but LSK or beddow is not the main point here, so ill just update the variable name